README.md

```diff
@@ -10,7 +10,7 @@ AMS does this through two components: **serializers** and **adapters**.
 Serializers describe _which_ attributes and relationships should be serialized.
 Adapters describe _how_ attributes and relationships should be serialized.
```

Comment:
reminder to my self to note that one cannot use a Serializer alone to get a hash or JSON.  It is the job of the adapter to compose the attributes-to-be-serialized of each object into the hash or JSON document.